balrogscript/submitter/cli.py

```diff
@@ -401,7 +399,7 @@ def run(self, platform, productName, appVersion, version, build_number, locale,
         api.update_build(
             data_version=data_version,
             product=productName, hashFunction=hashFunction,
-            buildData=json.dumps(merge_partial_updates(current_data, data)),
+            buildData=json.dumps(data),
```

Comment:
I'd normally put this in a separate request, but I wanted to avoid pulling in a new dep, just to remove it.

Reply:
For any platform+locale combination, we run one balrogscript task, that has both partials and completes. My assumption was that in the buildbot days, we would run this code multiple times, and so wanted to merge the results. Now the only time this will be hit for a platform+locale combination is if the task gets re-run, in which case we probably want *just* the new info (see [Bug 1501167](https://bugzilla.mozilla.org/show_bug.cgi?id=1501167) for example)

Reply:
Hm, this makes sense. I confused it with something else (balrog downloading existing blobs to enrich with new one for each locale).